### ISSUE TITLE
Registration password

### ISSUE THREAD

**Issue by conradludgate** [ellie/atuin#344]
We support two flows when registering:

Upfront (arguments):
```sh
atuin register -u "..." -e "..." -p "..."
```

Partial (prompt):
```sh
atuin register -u "..." -e "..."
Please enter password: ...
```

I think we should probably suggest the latter (it's uncommon to paste passwords into the command itself)

There's also the issue that the password prompt does not enable any proper password modes, meaning that the prompt will display the password as it's typed. Might not be what is expected

**Comment by conradludgate** [comment]
The same is true for login. Relevant code: https://github.com/ellie/atuin/blob/main/src/command/client/sync/login.rs#L45

**Comment by s0** [comment]
This can be closed since #424 is merged now right?
